BetterMD/elements/table.py

```
@@ -118,14 +120,25 @@ def to_rst(self, inner, symbol, parent, **kwargs):
             return "".center(width)
         return f"**{content}**".center(width)
 
+class ThRST(CustomRst):
+    def to_rst(self, inner, symbol, parent):
+        return " ".join([e.to_rst() for e in inner])
 
+class TBodyRST(CustomRst):
```

Ruff (0.8.2)

Redefinition of unused `TBodyRST` from line 99

(F811)